File: c#/web/WebApplication1/Entity/AreaTree.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Entity
{
    /// <summary>
    /// 地区树
    /// </summary>
    public class AreaTree
    {
        /// <summary>
        /// 地区名称
        /// </summary>
        public string name { set; get; }

        /// <summary>
        /// 地区值
        /// </summary>
        public string value { set; get; }

        /// <summary>
        /// 上级地区值
        /// </summary>
        public string parentValue { set; get; }

        /// <summary>
        /// 类型，0：省级、1：市级、2：区级、3：镇级
        /// </summary>
        public int type { set; get; }
    }
}

File: java/web/src/main/java/web/template/entity/common/AreaTree.java
package web.template.entity.common;
public class AreaTree {
	/**
	 * 地区名称
	 */
    public String name;

    /**
     * 地区值
     */
    public String value;

    /**
     * 上级地区值
     */
    public String parentValue;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getValue() {
		return value;
	}

	public void setValue(String value) {
		this.value = value;
	}

	public String getParentValue() {
		return parentValue;
	}

	public void setParentValue(String parentValue) {
		this.parentValue = parentValue;
	}
}